CRM/Aoservicelisting/Form/ProviderApplicationForm.php

```diff
@@ -97,11 +86,11 @@ public function setDefaultValues() {
   }
 
   public function buildQuickForm() {
-
     CRM_Core_Resources::singleton()->addStyleFile('biz.jmaconsulting.aoservicelisting', 'css/providerformstyle.css');
 
+    $attr = empty($this->organizationId) ? [] : ['readonly' => TRUE];
     $serviceListingOptions = [1 => E::ts('Individual'), 2 => E::ts('Organization')];
-    $listingTypeField = $this->addRadio('listing_type', E::ts('Type of Service Listing'), $serviceListingOptions);
+    $listingTypeField = $this->addRadio('listing_type', E::ts('Type of Service Listing'), $serviceListingOptions, $attr);
     $organizationNameField = $this->add('text', 'organization_name', E::ts('Organization Name'));
```

Comment:
@monishdeb I think we need to apply the same attr to Org Name and Org Email

Reply:
@JoeMurray I remember you mentioning on the ticket that in edit mode we should free org and individual name fields are you able to confirm which individual name fields we should be freezing?